nixos/modules/services/monitoring/grafana-agent.nix

```diff
@@ -48,19 +48,7 @@ in
         freeformType = settingsFormat.type;
       };
 
-      default = {
-        metrics = {
-          wal_directory = "\${STATE_DIRECTORY}";
-          global.scrape_interval = "5s";
-        };
-        integrations = {
-          agent.enabled = true;
-          agent.scrape_integration = true;
-          node_exporter.enabled = true;
-          replace_instance_label = true;
-        };
-      };
-
+      default = { };
```

Comment:
How could we do it so the default would appear in the docs as well?

Reply:
With `defaultText`. I'll update the PR.